.travis.yml

```diff
@@ -42,9 +42,6 @@ env:
   - REQUIREMENTS=lowest
   - REQUIREMENTS=release
   - REQUIREMENTS=devel
-  - REQUIREMENTS=lowest CLEANCSS=3.4.19
```

Comment:
we shouldn't leave this (or at least just one of the three) in order to test old and new clean css?